inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/preprocess.hpp

```diff
@@ -21,12 +22,27 @@ using preprocessParamsTuple = std::tuple<
         std::string>;                              // Device name
 
 class PrePostProcessTest : public testing::WithParamInterface<preprocessParamsTuple>,
-                        virtual public LayerTestsUtils::LayerTestsCommon {
+                           virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
`public ov::test::SubgraphBaseTest` supports both scenarios (static and dynamic). it would be better just to migrate `PrePostProcessTest` to the new base class

Reply:
Yes, but SubgraphBaseTest doesn't support automatic targetStaticShapes calculation if shape is static. I don't know if it is expected or not, so decided just to create new separate test